modules/sdp/LocalSdpMunger.js

```diff
@@ -28,77 +25,75 @@ export default class LocalSdpMunger {
     constructor(tpc, localEndpointId) {
         this.tpc = tpc;
         this.localEndpointId = localEndpointId;
-        this.audioSourcesToMsidMap = new Map();
-        this.videoSourcesToMsidMap = new Map();
-    }
-
-    /**
-     * Returns a string that can be set as the MSID attribute for a source.
-     *
-     * @param {string} mediaType - Media type of the source.
-     * @param {string} trackId - Id of the MediaStreamTrack associated with the source.
-     * @param {string} streamId - Id of the MediaStream associated with the source.
-     * @returns {string|null}
-     */
-    _generateMsidAttribute(mediaType, trackId, streamId) {
-        if (!(mediaType && trackId)) {
-            logger.error(`Unable to munge local MSID - track id=${trackId} or media type=${mediaType} is missing`);
-
-            return null;
-        }
-        const pcId = this.tpc.id;
-
-        return `${streamId}-${pcId} ${trackId}-${pcId}`;
     }
 
     /**
-     * Updates or adds a 'msid' attribute in the format '<endpoint_id>-<mediaType>-<trackIndex>-<tpcId>'
-     * example - d8ff91-video-0-1
-     * All other attributes like 'cname', 'label' and 'mslabel' are removed since these are not processed by Jicofo.
+     * Updates or adds a 'msid' attribute in the format. All other attributes like 'cname', 'label' and 'mslabel' are
```

Comment:
"Updates or adds a 'msid' attribute in the format." not sure what this means. I feel like a part was chopped and it no longer makes sense. Perhaps it should just say "Updates or adds a 'msid' attribute."

Reply:
Fixed the comments.